### DISCUSSION TITLE
Tile statistics

### DISCUSSION THREAD

**Discussion by valodo** [go-spatial/tegola#963] [Q&A]
Hello,

I am starting to use Tegola as tile server. I configured Tegola with different layers with mvt_postgis as provider and I would like to measure the time to generate the tiles and other statistics related to database.

Is there a configuration to achive this goal?

Thank you.

**Comment by ARolek** [comment]
@valodo tegola has observability support via Promethus. You can find the documentation at: https://github.com/go-spatial/tegola/blob/master/observability/prometheus/README.md. 

I know the crew at Wikimedia put together a Grafana dashboard using these metrics. @matheusmatos are you able to share a link to the dashboard for reference?
